### ISSUE TITLE
NuGet seems to be bugged for Toasty

### ISSUE THREAD

**Issue by mrousavy** [bengry/Toasty#4]
Can't Update your Package on NuGet Console or NuGet Manager, NuGet always throws an Exception that has no detail and can't be caught. I can't remove/update/install Toasty on [ImgurSniper](http://github.com/mrousavy/ImgurSniper).

**Comment by bengry** [comment]
I just started a new WPF project to test and installed Toasty (v1.0.0.3) without any issues.
Can you please describe a bit more what's the issue and environment - maybe you have some other issue that's surfaced by this, since Toasty has not dependencies.

**Comment by mrousavy** [comment]
I'm sorry, I didn't notice it was my fault, as I tried to add it do a Xamarin.Android project, and it didn't work, so I added it to a new Xamarin.Android Project, and it still didn't work. (The NuGet Manager just crashes and can't show any packages anymore, without being able to go back).

It was Android's fault tho, not yours, I reinstalled Windows as I can't open Visual Studio installers, etc. and now it works like a charm. Thank you for your effort!
